src/helper.js

```diff
@@ -0,0 +1,19 @@
+import { ERROR_MESSAGE } from "./constants/errorMessage";
+
+/**
+ * 계산 결과를 표현할 때 소수점 이하는 버림한다.
+ * @param (사칙연산)  - number
+ * @returns number without 소수점
+ */
+export const removeDecimalPoint = n => {
+  return Math.trunc(n);
+};
+
+/**
+ * 숫자는 한번에 최대 3자리 수까지만 다룰 수 있다. Boolean
+ * @param (사칙연산)  - number
+ * @returns boolean
+ */
+export const acceptThreeDigit = n => {
+  if (String(Math.abs(n)).length < 4) return true;
```

Comment:
- 아래 suggestion처럼 바로 boolean을 리턴하는 방식이 간결할 듯 싶습니다.
- 4라는 값이 이미 함수 안에 선언되어 해당 유틸의 사용 범위가 제한된다고 느껴지는데, 허용 가능한 자릿수를 인자로 받게 고려해보면 어떨까요?

```suggestion
  return String(Math.abs(n)).length < 4
```

Reply:
* 4라는 값이 이미 함수 안에 선언되어 해당 유틸의 사용 범위가 제한된다고 느껴지는데, 허용 가능한 자릿수를 인자로 받게 고려해보면 어떨까요? 

좋은 아이디어 같아요! 다음과 같은 형식일까요? 

```js
export const acceptThreeDigit = (n, num) => {
  return String(Math.abs(num)).length <= n;
}
```

Reply:
넵, 다만 인자의 네이밍은 보다 구체적으로 작성해주는 것이 좋겠습니다!